Velaptor/Graphics/Renderers/FontRenderer.cs

```diff
@@ -124,6 +131,93 @@ protected override void ShutDown()
         base.ShutDown();
     }
 
+    /// <summary>
+    /// Constructs a list of batch items from the given
+    /// <paramref name="charMetrics"/> to be rendered.
+    /// </summary>
+    /// <param name="textPos">The position to render the text.</param>
+    /// <param name="charMetrics">The glyph metrics of the characters in the text.</param>
+    /// <param name="font">The font being used.</param>
+    /// <param name="origin">The origin to rotate the text around.</param>
+    /// <param name="renderSize">The size of the text.</param>
+    /// <param name="angle">The angle of the text.</param>
+    /// <param name="color">The color of the text.</param>
+    /// <param name="atlasWidth">The width of the font texture atlas.</param>
+    /// <param name="atlasHeight">The height of the font texture atlas.</param>
+    /// <returns>The list of glyphs that make up the string as font batch items.</returns>
+    private static IEnumerable<FontGlyphBatchItem> ToFontBatchItems(
+        Vector2 textPos,
+        IEnumerable<GlyphMetrics> charMetrics,
+        IFont font,
+        Vector2 origin,
+        float renderSize,
+        float angle,
+        Color color,
+        float atlasWidth,
+        float atlasHeight)
+    {
+        var result = new List<FontGlyphBatchItem>();
+
+        var leftGlyphIndex = 0u;
+
+        foreach (var currentCharMetric in charMetrics)
+        {
+            textPos.X += font.GetKerning(leftGlyphIndex, currentCharMetric.CharIndex);
+
+            // Create the source rect
```

Comment:
Refactor all of these single line comments (if needed) to Creates, Calculates, Adjusts etc instead of Create, Calculate, Adjust. 
```suggestion
            // Creates the source rect
```